src/libraries/System.Reflection.MetadataLoadContext/tests/ILLink.Descriptors.xml

```diff
@@ -0,0 +1,3 @@
+<linker>
+  <assembly fullname="mscorlib" />
```

Comment:
Is this still necessary with the latest linker? We had a few fixes that went in to preserve facade assemblies.